src/assets/scss/_common.scss

```diff
@@ -1,6 +1,40 @@
+body {
+  font-family: verdana;
+  font-size: 16px;
+}
 .container {
   margin: 20px auto 20px auto;
   max-width: 500px;
+  .todo-form {
+    display: flex;
+    input {
+      flex-grow: 1;
+      padding: 10px;
+      margin-right: 5px;
+      font-size: 16px;
+    }
+    button {
+      font-size: 16px;
+    }
+  }
+  .list-item {
```

Comment:
Keep the styles in respective components.
ie. ``assets/scss/components/_list-item.scss``

Reply:
Done